wiki/Alphabet.md

```diff
@@ -0,0 +1,183 @@
+---
+title: History and replacement of Bio.Alphabet
+permalink: wiki/Alphabet
+layout: wiki
+tags:
+ - Wiki Documentation
+---
+
+# Introduction
+
+This page is intended to help people updating existing code using Biopython to
+cope with the removal of the ``Bio.Alphabet`` module in Biopython 1.78
+(September 2020).
+
+The objects from `Bio.Alphabet` had two main uses:
+* Record the molecule type (DNA, RNA or protein) of a sequence,
+* Declare the expected characters in a sequence, alignment, motif, etc.
+
+# Motivation
+
+There were drawbacks to the twenty year old design. In particular how the
+`AlphabetEncoder` class (used to add information about gap or stop symbols)
+was overly complicated, and made even determining the molecule type difficult.
+Taking the consensus of multiple alphabet objects (for example during string
+addition) was also complicated. While you could specify a strict alphabet like
+unambiguous IUPC DNA for a sequence, this did not enforce that only letters
+used were `A`, `C`, `G` and `T`.
+
+# Code changes
+
+After failing to reach any consensus on how to improve or replace the existing
```

Comment:
I don't think a lack of consensus was a major issue. The decision to remove alphabets was made after we called for a Biopython Enhancement Proposal to define unambiguously the purpose of alphabets. One person volunteered, and that person in the end came to the conclusion that alphabets were ill-advised. Based on that, and on the fact that nobody else stepped up to define alphabets, we decided to remove them.

Reply:
Would you prefer this wording:

"With no clear proposal on how to improve or replace the existing system, it was agreed to remove it."

Reply:
ok